Game.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Block from './Block';
+
+function Game() {
+
+  const [isXcurrentPlayer, setNextPlayer] = React.useState(true);
+
+    function changePlayer(){
+    setNextPlayer(prevState => {
+      if(prevState) return false; 
+      else return true;
+        })
+    }
+
+    function draw(e){        
+      if(e.target.textContent !== "") return;
+      isXcurrentPlayer? e.target.textContent = "X": e.target.textContent = "O";
+      checkForWinner();
+      changePlayer();
+      }
+
+  const boxfields =  document.getElementsByClassName("box--field");
+  //its collection so we must use .textContent
+
+  const winCombinations = [
+    [0, 1 ,2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 9],
+    [0, 4, 8],
+    [2, 4, 6],
+];
+
+
+    function checkForWinner(){
+        for(let i=0; i<8; i++){
```

Comment:
for loops are not that much used since ES6.
You can do: winCombinations.forEach((combination) => { ... } )